data/etc/plinth/modules-enabled/diaspora

```diff
@@ -1,2 +0,0 @@
-plinth.modules.diaspora
```

Comment:
Ok, this change looks fine then.

A pull request in Alioth is just a bug report to Debian BTS, with a patch attached (git format-patch). We could probably use a tutorial on the wiki.